libraries/src/Client/FtpClient.php

```diff
@@ -291,10 +291,7 @@ public function connect($host = '127.0.0.1', $port = 21)
     {
         $errno = null;
         $err   = null;
-
-// var_dump($this->_conn);
-// die("dfg");
-
+        
         // If already connected, return
```

Comment:
```suggestion

        // If already connected, return
```
Remove spaces from blank line to fix failing PHP code style check.